src/redturtle/prenotazioni/restapi/serializers/adapters/prenotazioni_folder.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+from copy import deepcopy
+
+from plone.restapi.interfaces import ISerializeToJson
+from plone.restapi.serializer.dxcontent import SerializeFolderToJson
+from zope.component import adapter
+from zope.interface import implementer
+
+from redturtle.prenotazioni.content.prenotazioni_folder import IPrenotazioniFolder
+from redturtle.prenotazioni.interfaces import IRedturtlePrenotazioniLayer
+
+
+@implementer(ISerializeToJson)
+@adapter(IPrenotazioniFolder, IRedturtlePrenotazioniLayer)
+class PrenotazioniFolderSerializer(SerializeFolderToJson):
+    def __call__(self, *args, **kwargs):
+        res = super().__call__()
+
+        for index, type in enumerate(deepcopy(res.get("booking_types", {}))):
+            if type.get("hidden"):
+                del res["booking_types"][index]
```

Comment:
```suggestion
        res["booking_types"] = [t for t in res["booking_types"] if not t.get("hidden")]
```

Reply:
Solved, thanks for suggestions

Reply:
Ah ok, I could even commit the suggestion directly